client/vat/form/tasks/vat-number-task.tsx

```diff
@@ -66,6 +80,12 @@ const getVatTaxIDRequirementHint = () => {
 
 const getVatTaxIDValidationHint = () => {
 	switch ( wcpaySettings.accountStatus.country ) {
+		case 'AU':
+			// https://abr.business.gov.au/Help/AbnFormat
+			return __(
+				'11-digit number, for example 51 824 753 556.',
```

Comment:
I think we should add an example rather than a valid number here. This valid number is actually Australian Tax Office, we might not suggest it, right? 😬 
```suggestion
				'11-digit number, for example 12 345 678 901.',
```